peps/pep-0729.rst

```diff
@@ -0,0 +1,414 @@
+  such as runtime type checking, static compilation, IDE support, and documentation.
+  The Typing Council should consider all of these use cases when making decisions,
+  and be open to supporting additional use cases as they come up.
+* **Usable**: The type system should be easy to use for Python developers. It
+  should be ergonomic to write well-typed Python code that is accepted by type
+  checkers. There should be good documentation for the type system.
+* **Stable**: As the type system matures, users should be able to rely on their
+  typed code continuing to work and be able to trust their mental model for the
+  type system. Changes should be made with care and in a way
+  that minimizes disruption. Nevertheless, the type system should be able to
+  evolve, and type checkers need not be held to the same strict compatibility
+  standards as the Python language itself.
+
+Operations and process
+----------------------
+
+The council would have three members, comprised of prominent community members,
+such as Python core developers and maintainers of major type checkers. The members
+should include people affiliated with a variety of projects related to type checking,
+which may include type checkers, CPython, typeshed, or other projects.
+
+The Steering Council would appoint the initial Typing Council. There is no term
+limit for council members. Council members may resign their position at any time.
+There is a norm that at least one person will voluntarily resign from the
+Typing Council each year.
+
+To determine replacements, nominations will be collected from the typing
+community. Self-nominations are allowed. The existing Typing Council will then decide
+the replacement member(s) from the nominees. The expectation is that this would
+be done by fiat, but the Typing Council can choose a replacement by any means
+they see fit, including a vote.
+
+The Typing Council remains accountable to the Steering Council. At any point,
+for any reason, the Steering Council could (publicly or privately) make a
+specific change or request a non-specific change to the composition of the
+Typing Council.
+
+We acknowledge that this is a not particularly democratic structure and puts
+a lot of faith in the Typing Council. However, the Python community has a long
+history of success with not particularly democratic structures! We believe
+self-governance, cycling of membership, and accountability to the
+Steering Council will be sufficient to ensure that the Typing Council is meeting
+the needs of the community.
+
+The council would operate primarily through reviews of GitHub PRs. Regular
+meetings are likely not necessary, but the council may set up video calls, a
+private chat, or whatever other mechanism they decide upon internally.
+
+The council should aim for transparency, posting all decisions publicly, with a
+rationale if possible.
+
+Members of the council are eligible to sponsor PEPs.
```

Comment:
This  is a deviation from the current process around sponsorship. Should it be noted in the relevant PEP?

Reply:
Yes, this is intended as a change proposal, so if this PEP is accepted, the relevant PEP (PEP 1 I think) should be amended. Let's make that explicit:

```suggestion
Members of the council will be eligible to sponsor PEPs. If this PEP is accepted,
:pep:`1` should be amended to note this fact.
```